package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon server/server.js",
+    "start": "nodemon --watch server --exec 'node -r esm server/server.js'",
```

Comment:
nodemon, if not looking inside `root` or `/src` needs to have a `watch` argument. The `--exec` is telling nodemon to execute a special script (in this case, `node -r esm server/server.js`) rather than just `node`